### PR TITLE
Enable log pretty-printing

### PR DESCRIPTION
**Pull request by sbly**
Calling this function enables all the logging-related options, including pretty-printing, log levels, formatters, and log rotation.

### CODE REVIEW

File: oz/core/actions.py [hunk 1]
```diff
@@ -65,6 +65,13 @@ def init(project_name):
 def server():
     """Runs the server"""
 
+    if oz.settings["logging"]:
```
Comment (ysimonson) [inline oz/core/actions.py]: I don't think this should be necessary, as tornado handles it for you. We just need to add an option for `logging` under https://github.com/dailymuse/oz/blob/develop/oz/core/options.py

Comment (sbly) [inline oz/core/actions.py]: The logging options are automatically available due to [this function](http://www.tornadoweb.org/en/stable/log.html#tornado.log.define_logging_options). It wasn't working when I tried just setting the logging option in our config.py, but after searching through the codebase it looks like I can just call this http://www.tornadoweb.org/en/stable/log.html#tornado.log.enable_pretty_logging

**Comment by sbly** [comment]
@ysimonson You're right, it could be simplified

File: oz/core/actions.py (hunk 1 above)
Comment (ysimonson) [inline oz/core/actions.py]: that should be a separate thing. `enable_pretty_logging` should just enable ANSI colors in the terminal.

Comment (sbly) [inline oz/core/actions.py]: That's not what it does though. You can check the source code (there's a link in the documentation).

Comment (ysimonson) [inline oz/core/actions.py]: fair enough, though I don't see it doing anything with setting logging level

Comment (sbly) [inline oz/core/actions.py]: `logger.setLevel(getattr(logging, options.logging.upper()))`

Comment (sbly) [inline oz/core/actions.py]: (I agree this is extremely poorly communicated, which is why we never noticed until now)

Comment (ysimonson) [inline oz/core/actions.py]: ah! good catch.

**Pull request merged**